### ISSUE TITLE
RC Test: Molecules and Light 1.5.0-rc.2

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#513]
@KatieWoe , @ariel-phet, @terracoda @emily-phet, molecules-and-light/1.5.0-rc.2 is ready for RC testing.
This release candidate follows the dev test in https://github.com/phetsims/QA/issues/503.

This version includes accessibility features including screen reader support, sound, iOS VoiceOver, and
magnification, so please include the "Accessibility" section in this issue.

Also, this version includes the new IE11 content in https://github.com/phetsims/chipper/issues/954. @chrisklus 
mentioned extra things should be checked for this release. @chrisklus can you please add any comments
in this issue?

Document issues in https://github.com/phetsims/molecules-and-light/issues and link to this issue.

Assigning to @ariel-phet for prioritization.

<details>
<summary><b>General RC Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Test all query parameters on all platforms. (See [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md)
for a list of query parameters.)
- Download HTML on Chrome and iOS.
- Make sure the iFrame version of the simulation is working as intended on all platforms.
- Make sure the XHTML version of the simulation is working as intended on all platforms.
- Complete the test matrix.
- Don't forget to make sure the sim works with Legends of Learning.
- Check [this](https://docs.google.com/spreadsheets/d/1umIAmhn89WN1nzcHKhYJcv-n3Oj6ps1wITc-CjWYytE/edit#gid=0) LoL
spreadsheet and notify AR or AM if it not there.
- If this is rc.1 please do a memory test.
- When making an issue, check to see if it was in a previously published version
- Try to include version numbers for browsers
- If there is a console available, check for errors and include them in the Problem Description.
- As an RC begins and ends, check the sim repo. If there is a maintenance issue, check it and notify developers if
there is a problem.

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-rc.2/phet/molecules-and-light_all_phet.html)**
- **[iFrame](https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-rc.2/phet/molecules-and-light_all_iframe_phet.html)**
- **[XHTML](https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-rc.2/phet/xhtml/molecules-and-light_all.xhtml)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/1OUzxwzBS3Bpws1zu5sM5st-b7dNzN-f08EHxvIW4qoA/edit#gid=1253450703)**
- **[Legends of Learning Harness](https://developers.legendsoflearning.com/public-harness/index.html?startPayload=%7B%22languageCode%22%3A%22en%22%7D)**

<hr>

</details>

<details>
<summary><b>Accessibility (a11y) RC Test</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found below.
- Make sure the a11y feature doesn't negatively affect the sim in any way.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
  - Test all keyboard navigation inputs (if applicable).
  - Test all forms of input with a screen reader (if applicable).
- Test magnification. Magnification is not expected to work while the screen reader is enabled. You should be able to zoom in and move the sim with:
  - Keyboard input.
  - Touch screen input.
  - Mouse input (ctrl + wheel, click and drag).

<h3>Link(s)</h3>

- **[a11y View](https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-rc.2/phet/molecules-and-light_a11y_view.html)**
- **[Simulation](https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-rc.2/phet/molecules-and-light_all_phet.html)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/1OUzxwzBS3Bpws1zu5sM5st-b7dNzN-f08EHxvIW4qoA/edit#gid=1253450703)**

<h3>Screen Readers</h3>

This sim supports screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to
screen readers. If you simply need a refresher on screen readers, please consult the [QA Book](link), which should have
all of the information you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the
a11y view before opening the simulation. Once you've done that, open the simulation and make sure alerts and
descriptions work as intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- [x] Windows 10 + Latest Chrome + Latest JAWS
- [x] Windows 10 + Latest Firefox + Latest NVDA
- [x] macOS + Safari + VoiceOver
- [x] iOS + Safari + VoiceOver (only if specified in testing issue)

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in
[this Google Document](https://drive.google.com/open?id=12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw). If you find a
screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a
screen reader.

<h3>Final Requests</h3>

- [ ] If this sim is in [this list](https://github.com/phetsims/perennial/blob/master/data/accessibility), make an
issue in [website](https://github.com/phetsims/website) to ask if PhET research page links need updating. Please assign
to @terracoda and @emily-phet.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>

  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary>

  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>

  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>

  We typically assign the developer who opened the issue in the QA repository.

  </details>

  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>

  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by ariel-phet** [comment]
@KatieWoe current new top testing priority

**Comment by KatieWoe** [comment]
Memory test results:
![memtest](https://user-images.githubusercontent.com/41024075/87174229-1c104c80-c294-11ea-9627-f1a837551f72.png)

**Comment by chrisklus** [comment]
@KatieWoe - could you please spot check test the "IE Error" page from https://github.com/phetsims/chipper/issues/954 as part of this issue? It should just involve making sure that the following page appears for every sim deliverable when launched on IE11 (with dev tools closed). Please let me know if you have any questions and please report any problems as additional comments in https://github.com/phetsims/chipper/issues/954. Thanks!

<img width="500" alt="Screen Shot 2020-07-16 at 3 13 52 PM" src="https://user-images.githubusercontent.com/5618316/87712689-0f4d9600-c777-11ea-83cd-e24dc880d7ac.png">

**Comment by KatieWoe** [comment]
https://github.com/phetsims/QA/issues/513#issuecomment-659615883 looks ok for the sim, the iframe, and the xhtml. @chrisklus

**Comment by terracoda** [comment]
@KatieWoe, is it possible to also test the message with JAWS and NVDA. It's possible you cannot due to versioning issues.

I am just curious.

**Comment by KatieWoe** [comment]
I can check it out. It didn't occur to me since we only test Jaws and Nvda with chrome and firefox respectively, but I'll take a look really quick.

**Comment by KatieWoe** [comment]
Looks ok with both @terracoda

**Comment by chrisklus** [comment]
Awesome, thanks much!

**Comment by KatieWoe** [comment]
QA is done. I skipped the iOS 9 and 10 for time reasons as those platforms are not supported and used for performance testing.

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe and team. There are 4 remaining issues listed above which have fixes in the release branch ready for verification.  Ill prepare a spot check for those.
